components/schema-migrator/migrations/director/20220127152623_add_system_status_field_to_applications.up.sql

```diff
@@ -0,0 +1,6 @@
+BEGIN;
+
+ALTER TABLE applications
+    ADD COLUMN system_status TEXT;
```

Comment:
Consider adding some length restrictions, because `TEXT` is unrestricted:

```sql
ALTER TABLE applications
	ADD CONSTRAINT system_status_len CHECK (char_length(system_status) <= 123);
```